website/docs/docs/quickstarts/dbt-cloud/databricks-qs.md

```diff
@@ -0,0 +1,286 @@
+    </div>
+
+7. Update the Table Attributes at the top:
+
+    - <b>data_catalog</b> = hive_metastore
+    - <b>database</b> = default 
+    - <b>table</b> = jaffle_shop_customers
+    - Make sure that the column data types are correct. The way you can do this is by hovering over the datatype icon next to the column name. 
+        - <b>ID</b> = bigint
+        - <b>FIRST_NAME</b> = string
+        - <b>LAST_NAME</b> = string
+
+    <div style={{maxWidth: '600px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/jaffle_shop_customers_upload.png" title="Load jaffle shop customers" />
+    </div>
+
+8. Click **Create** on the bottom once you’re done. 
+
+9. Now let’s do the same for `Jaffle Shop Orders` and `Stripe Payments`. 
+
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/jaffle_shop_orders_upload.png" title="Load jaffle shop orders" />
+    </div>
+
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/stripe_payments_upload.png" title="Load stripe payments" />
+    </div>
+    
+10. Once that's done, make sure you can query the training data.  Navigate to the `SQL Editor` through the left hand menu.  This will bring you to a query editor.
+11. Ensure that you can run a `select *` from each of the tables with the following code snippets. 
+
+    ```sql
+    select * from default.jaffle_shop_customers
+    select * from default.jaffle_shop_orders
+    select * from default.stripe_payments
+    ```
+
+    <div style={{maxWidth: '400px'}}>
+    <Lightbox src="/img/databricks_tutorial/images/query_check.png" title="Query Check" />
+    </div>
+
+12. To ensure any users who might be working on your dbt project has access to your object, run this command.
+
+    ```sql 
+    grant all privileges on schema default to users;
+    ```
+
+## Connect dbt Cloud to Databricks
+
+There are two ways to connect dbt Cloud to Databricks. The first option is Partner Connect, which provides a streamlined setup to create your dbt Cloud account from within your new Databricks trial account. The second option is to create your dbt Cloud account separately and build the Databricks connection yourself (connect manually). If you are looking to get started quickly, dbt Labs recommends using Partner Connect. If you are looking to customize your setup from the very beginning and gain familiarity with the dbt Cloud setup flow, we recommend connecting manually.
+
+If you want to use Partner Connect, refer to [Connect to dbt Cloud using Partner Connect](https://docs.databricks.com/partners/prep/dbt-cloud.html#connect-to-dbt-cloud-using-partner-connect) in the Databricks docs for instructions. 
+
+If you want to connect manually, refer to [Connect to dbt Cloud manually](https://docs.databricks.com/partners/prep/dbt-cloud.html#connect-to-dbt-cloud-manually) in the Databricks docs for instructions.
```

Comment:
For snowflake, we provided instructions inline. Any reason we haven't for Databricks?

Reply:
good question! i couldn't find something similar in the Snowflake docs. i did see this (which i linked to in our qs) https://quickstarts.snowflake.com/guide/data_teams_with_dbt_cloud/#3 but it loads different sample data, not Jaffle Shop, so couldn't simply link out to it